src/components/Upgrade/HighlightScreen.js

```diff
@@ -193,7 +200,7 @@ HighlightScreen.propTypes = {
   enterProgress: PropTypes.object,
   onUpgrade: PropTypes.func.isRequired,
   showProgress: PropTypes.object,
-  status: PropTypes.oneOf(['entering', 'leaving']),
+  state: PropTypes.string,
```

Comment:
Should we leave this as a `oneOf`?

Reply:
I decided not to so we don’t have to keep it in sync with react-spring (since it’s directly coming from it now). Do you think we should list the react-spring states here?

Reply:
I think we might as well, just in case it accidentally moves underneath us without us noticing :).